MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -37,6 +39,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		tableModel = cellControllers
 	}
 
+	func display(_ viewModel: FeedErrorViewModel) {
+		if let message = viewModel.errorMessage {
+			errorView?.show(message: message)
+		} else {
+			errorView?.hideMessage()
```

Comment:
No need to put question marks after `errorView` as it is has force unwrapped type (`ErrorView!`), please remove the question marks

```suggestion
			errorView.show(message: message)
		} else {
			errorView.hideMessage()
```

Reply:
Done